### PR TITLE
Add `enable_stealth` and `disable_stealth` methods

### PR DESCRIPTION
**Pull request by YunusovSamat**
# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

File: botx/bot/bot.py [hunk 1]
```diff
@@ -329,12 +371,31 @@ async def create_chat(
 
         return botx_api_chat_id.to_domain()
 
+    async def disable_stealth(self, bot_id: UUID, chat_id: UUID) -> None:
+        """Disable stealth model. Hides all messages that were in stealth.
+
+        **Arguments:**
+
+        * `bot_id: UUID` - Bot which should perform the request.
+        * `chat_id: UUID` - ID of chat where stealth should be disable.
```
Comment (Nimond) [inline botx/bot/bot.py:380]: "Target chat id." plz

File: botx/bot/bot.py [hunk 2]
```diff
@@ -287,6 +296,39 @@ async def remove_users_from_chat(
...
+        disable_in_web_client: bool = False,
+        ttl_after_read: Optional[int] = None,
+        total_ttl: Optional[int] = None,
+    ) -> None:
+        """Enable stealth mode. After the expiration of the time all messages will be hidden.
+
+        **Arguments:**
+
+        * `bot_id: UUID` - Bot which should perform the request.
+        * `chat_id: UUID` - ID of chat where stealth should be enabled.
```
Comment (Nimond) [inline botx/bot/bot.py:312]: "Target chat id." plz

**Review by Nimond — CHANGES_REQUESTED** [review]

File: botx/client/chats_api/disable_stealth.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    async def execute(self, payload: BotXAPIDisableStealthRequestPayload) -> None:
+        path = "/api/v3/botx/chats/stealth_disable"
+
+        response = await self._botx_method_call(
+            "POST",
+            self._build_url(path),
+            json=payload.jsonable_dict(),
+        )
+
+        self._extract_api_model(BotXAPIDisableStealthResponsePayload, response)
```
Comment (Nimond) [inline botx/client/chats_api/disable_stealth.py:51]: Rebase from next, this method now have the other name

**Review by Nimond — CHANGES_REQUESTED** [review]

**Review by Nimond — APPROVED** [review]

File: botx/bot/bot.py [hunk 4]
```diff
@@ -290,6 +299,57 @@ async def remove_users_from_chat(
...
+        disable_in_web_client: bool = False,
+        ttl_after_read: Optional[int] = None,
+        total_ttl: Optional[int] = None,
+    ) -> None:
+        """Enable stealth mode. After the expiration of the time all messages will be hidden.
+
+        **Arguments:**
+
+        * `bot_id: UUID` - Bot which should perform the request.
+        * `chat_id: UUID` - Target chat id..
```
Comment (lxmnk) [inline botx/bot/bot.py:315]: Unnecessary dot.

**Pull request merged**